src/java.naming/share/classes/javax/naming/directory/InitialDirContext.java

```diff
@@ -181,69 +181,115 @@ public Attributes getAttributes(Name name, String[] attrIds)
         return getURLOrDefaultInitDirCtx(name).getAttributes(name, attrIds);
     }
 
+    /**
+     * @throws  AttributeModificationException {@inheritDoc}
+     */
```

Comment:
Isn't `{@inheritDoc}` needed in this case to preserve the method & params  description?

Reply:
Elements of a method declaration such as formal and type parameters as well as exceptions in the `throws` clause implicitly inherit descriptions, unless those elements are described in that method declaration. The same goes for the main description of a method: unless it exists in the method declaration, it is inherited.

To sum up, those descriptions you ask about are implicitly inherited. No additional javadoc markup is needed.